app/features/ledger/ConcordiumLedgerClientMain.ts

```diff
@@ -83,12 +84,12 @@ export default class ConcordiumLedgerClientMain {
         return getSignedPublicKey(this.transport, path);
     }
 
-    getIdCredSec(identity: number): Promise<Buffer> {
-        return getIdCredSec(this.transport, identity);
+    getPrivateKeySeeds(identity: number): Promise<PrivateKeySeeds> {
+        return getPrivateKeySeeds(this.transport, identity);
     }
 
-    getPrfKey(identity: number): Promise<Buffer> {
-        return getPrfKey(this.transport, identity);
+    getPrfKey(identity: number, forRecovery?: boolean): Promise<Buffer> {
```

Comment:
Optional boolean parameters are bad as they hold 3 values instead of 2. On top of that it would be better to completely avoid the boolean parameter and instead refactor it into 2 separate methods, one for getting PRF key for decryption and one for recovery. So we get
```
getPrfKeyDecrypt(identity: number)
getPrfKeyRecovery(identity: number)
```